packages/suite/src/hooks/wallet/useCoinmarketSavingsSetup.ts

```diff
@@ -106,7 +106,7 @@ export const useSavingsSetup = ({
     ]);
 
     // NOTE: There is only one fiat currency per provider.
-    const fiatCurrency = selectedProvider?.tradedFiatCurrencies[0];
+    const fiatCurrency = selectedProvider?.tradedFiatCurrencies[0].toLocaleLowerCase();
```

Comment:
```suggestion
    const fiatCurrency = selectedProvider?.tradedFiatCurrencies[0];
```

Why `Locale`
Are you sure it does not break Invity api? I would lowercase it just for `localCurrency` which is used in `updateFiatRatesThunk `

Reply:
Hmm good point, will do it just for our selectors to be sure we don't break anything.